### PR TITLE
Handle message size more than 4096 characters

### PR DESCRIPTION
**Pull request by nishitm**
- Since Telegram sendMessage imposes a limit of 4096 characters, Dividing the message into the size of 4096 chunks and sending it multiple times
- Solving issue #7

### CODE REVIEW

File: src/main/java/io/github/trashemail/Configurations/TelegramConfg.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
 public class TelegramConfg{
     private String botToken;
     private String url;
+    private int size=4096;
```
Comment (r0hi7) [inline src/main/java/io/github/trashemail/Configurations/TelegramConfg.java:11]: I really like the idea of keeping this stuff in config.

File: src/main/java/io/github/trashemail/Telegram/SendTelegramMessage.java [hunk 2]
```diff
@@ -29,23 +31,32 @@ public void sendMessage(String message, String chatId){
...
 
-        HttpEntity<MultiValueMap<String, String>> request = new HttpEntity<MultiValueMap<String, String>>(data, headers);
+        	HttpEntity<MultiValueMap<String, String>> request = new HttpEntity<MultiValueMap<String, String>>(data, headers);
 
-        RestTemplate restTemplate = new RestTemplate();
-        ResponseEntity response = restTemplate.postForEntity(
-                telegramURI,
-                request,
-                String.class);
+        	RestTemplate restTemplate = new RestTemplate();
```
Comment (r0hi7) [inline src/main/java/io/github/trashemail/Telegram/SendTelegramMessage.java:49]: `RestTemplate` can be auto-wired, let Spring maintain this dependency. What you can do is, at the class level, 
```java
@Autowired
RestTemplate restTemplate;
```
And in this use case, you can remove the object creation at this particular line and then `restTemplate` can be used the normal way.
Do let me know if you have any issues getting this, what I can do it simply merge this to current master and then work on this in next commit.

File: src/main/java/io/github/trashemail/Configurations/TelegramConfg.java [hunk 3]
```diff
@@ -24,4 +25,12 @@ public String getBotToken() {
     public void setBotToken(String botToken) {
         this.botToken = botToken;
     }
+
+    public int getSize() {
+    	return size;
```
Comment (r0hi7) [inline src/main/java/io/github/trashemail/Configurations/TelegramConfg.java:30]: Can you add tabs instead of two spaces?

Comment (nishitm) [reply src/main/java/io/github/trashemail/Configurations/TelegramConfg.java:30]: Sure

**Pull request merged**